### PR TITLE
Improve project/milestone list

### PR DESCRIPTION
**Pull request by yp05327**
Project list
Before:
image
After:
image

Milestone List
Before:
image
After:
image

ps: It seems that `project list` is a copy of `milestone list`, and we have made some changes to `milestone list`.
Then we need to sync the changes to `project list`.

### CODE REVIEW

**Comment by yp05327** [comment]
image
The icons here seems strange 😕 
Maybe we need also fix it in milestone list.

File: templates/projects/list.tmpl [hunk 1]
```diff
@@ -38,7 +38,11 @@
 		<div class="milestone list">
```
Comment (silverwind) [inline templates/projects/list.tmpl:38]: Yes, it's a blatant copy as seen here. Possible to fix these to `projects`? CSS may need adjustment.

**Comment by silverwind** [comment]
> image The icons here seems strange 😕 Maybe we need also fix it in milestone list.

Yes, vertical center via a wrapper element possibly and `gt-df gt-ac`

**Comment by wxiaoguang** [comment]
> > image The icons here seems strange 😕 Maybe we need also fix it in milestone list.
> 
> Yes, vertical center via a wrapper element possibly and `gt-df gt-ac`

After #24476 , the svg icons should be display:inline-block + vertical-align:middle , by default.

**Comment by silverwind** [comment]
> display:inline-block + vertical-align:middle

I recall this is not a general solution and may still cause misalignment.

**Comment by wxiaoguang** [comment]
> may still cause misalignment depending on icon size

Do the best to use the same icon side as the font size everywhere. That's why font-awesome could win in such case.

**Comment by yp05327** [comment]
> > image The icons here seems strange 😕 Maybe we need also fix it in milestone list.
> 
> Yes, vertical center via a wrapper element possibly and `gt-df gt-ac`

I fixed this by use `gt-ml-3`. Maybe it is not the best/correct way.
@silverwind @wxiaoguang 
You are welcome to push commits/PR to fix it in the best/correct way. 😄

**Comment by silverwind** [comment]
> ps: the word size is 19, so I also adjusted the svg size.

It looks still somewhat broken. Should check how issue list renders its icons, we don't have such issues there I suppose. Maybe I will also check it later.

**Comment by yp05327** [comment]
It seems that this is fixed in #25315

**Pull request closed without merging**